core/test/financial_templates/ExpiringMultiParty.js

```diff
@@ -30,6 +31,18 @@ contract("ExpiringMultiParty", function(accounts) {
       sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
       disputerDisputeRewardPct: { rawValue: toWei("0.1") }
     };
+
+    identifierWhitelist = await IdentifierWhitelist.new({ from: accounts[0] });
```

Comment:
Since we're going to have to start using some `.deployed()` contracts, why not just do that for all of the singleton contracts? You won't have to write this deployment command and you wouldn't have to add it to the finder. You could also get rid of most of the changes below since they're just adding the identifier whitelist to the finder.

Reply:
This should also be part of a larger refactor as this does not only effect this one contract's tests but rather `liquidatable` and `pricelessPositionManager`. I have created an issue to address this here: #914